PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -11,8 +11,31 @@
 import org.json.JSONObject;
 
 /**
- * Used to create and tokenize PayPal accounts. For more information see the
- * <a href="https://developers.braintreepayments.com/guides/paypal/overview/android/">documentation</a>
+ * PayPalClient enables you to obtain permission to charge your customers' PayPal accounts by presenting the PayPal website.
+ *
+ * To make PayPal available, you must ensure that PayPal is enabled in your Braintree control panel.
+ * See our [online documentation](https://developers.braintreepayments.com/guides/paypal/overview/android/) for
+ * details.
+ *
+ * This class supports two basic use-cases: Vault and Checkout. Each of these involves variations on the
+ * user experience as well as variations on the capabilities granted to you by this authorization.
+ *
+ * The *Vault* option uses PayPal's future payments authorization, which allows your merchant account to
+ * charge this customer arbitrary amounts for a long period of time into the future (unless the user
+ * manually revokes this permission in their PayPal control panel.) This authorization flow includes
+ * a screen with legal language that directs the user to agree to the terms of Future Payments.
```

Comment:
```suggestion
 * a screen with legal language that directs the user to agree to the terms of the billing agreement.
```

I think `future payments` is the old terminology for billing agreement AKA vault.